wiki/PlayersManual.md

```diff
@@ -350,25 +350,35 @@ Trading panel: | &nbsp;
 :--- | :---
 Buy | +, Return, or Space
 Sell | -, Delete, or Backspace
-Buy / sell 5x | Shift
-Buy / sell 20x | Control / Command
-Buy / sell 500x | Option / Alt
 Buy / sell all | Shift + B / S
 View commodity price map | Map key
 
-The multiplier modifiers (Shift = 5x, Command / Control = 20x, and Option / Alt = 500x) also work in the hiring panel, the shipyard, and the outfitter. And, they can be combined together - for example, Shift + Control to buy or sell 100 items at a time.
+Multipliers: | &nbsp;
+:--- | :---
+5x | Shift
+20x | Control / Command
+500x | Option / Alt
```

Comment:
```suggestion
100x | Command & Shift
500x | Option / Alt
```
Something like this as well, maybe?

Reply:
Yeah, except my Windows/Linux keyboard doesn't have a "Command" key... could/should we add a Mac/PC column?

Reply:
I'm not sure that would be necessary. How about this?
```suggestion
100x | Command / Control & Shift
500x | Option / Alt
```

Reply:
Now I feel like it needs parentheses to clarify the order of operations...
I'm not looking at the table right now I'm only looking at the Delta in the code so maybe it looks better in the table but this could be interpeeted as (command) or (control+shift), or (command or control) + shift

Reply:
See updates, added new combinations table